lms/static/sass/features/_course-experience.scss

```diff
@@ -17,11 +20,11 @@
   .message-content {
     @include margin(0, 0, $baseline, $baseline);
 
+    flex-grow: 1;
```

Comment:
<img width="891" alt="screen shot 2017-10-06 at 2 10 04 pm" src="https://user-images.githubusercontent.com/3861052/31292118-2de9b720-aaa0-11e7-8809-5d36273fc5d8.png">

Reply:
Whoops! Let me look at that!

Reply:
Fixed